<launch>
    <arg name="robot1_name" default="pioneer"/>
    <arg name="robot2_name" default="pioneer_bot"/>

    <node name="move_base_checker_node" type="move_base_checker_node" pkg="uml_hri_nerve_navigation" output="screen" required="true">
        <param name="robot1_name" value="$(arg robot1_name)"/>
        <param name="robot2_name" value="$(arg robot2_name)"/>
    </node>

</launch>